library/common/config_template.cc

```diff
@@ -28,6 +28,46 @@ R"(
             - {{ domain }}
       sni: {{ domain }}
     type: LOGICAL_DNS
+  - name: base_wlan # Note: the direct API depends on the existence of a cluster with this name.
+    connect_timeout: {{ connect_timeout_seconds }}s
+    dns_refresh_rate: {{ dns_refresh_rate_seconds }}s
+    dns_lookup_family: V4_ONLY
+    lb_policy: ROUND_ROBIN
+    load_assignment:
+      cluster_name: base
```

Comment:
Should this be `base_wlan`? Same with `base_wwan` below

Reply:
good catch, yes it should